docs/docsite/rst/dev_guide/developing_inventory.rst

```diff
@@ -1,21 +1,220 @@
 .. _developing_inventory:
 
-Developing Dynamic Inventory Sources
-====================================
+Developing Dynamic Inventory
+============================
 
 .. contents:: Topics
    :local:
 
-As described in :ref:`dynamic_inventory`, Ansible can pull inventory information from dynamic sources, including cloud sources. You can also create a new dynamic inventory provider by creating a script or program that can output JSON in the correct format when invoked with the proper arguments. There is no restriction on the language used for creating a dynamic inventory provider.
+As described in :ref:`dynamic_inventory`, Ansible can pull inventory information from dynamic sources,
+including cloud sources, using the supplied :ref:`inventory plugins <Inventory_Plugins>`.
+If the source you want is not currently covered by existing plugins, you can create your own as with any other plugin type.
+
+In previous versions you had to create a a script or program that can output JSON in the correct format when invoked with the proper arguments.
+This is still supported via the :ref:`script inventory plugin <script_inventory>` and there is no restriction on the programming language used.
```

Comment:
Use active voice wherever possible - instead of "this is still supported . . . " I'd say "You can still write a script for dynamic inventory management using the :ref:`script inventory plugin` in any programming language you choose."